h/static/scripts/permissions-service.coffee

```diff
@@ -68,38 +72,17 @@ class Permissions
 
   # Creates access-level-control object list
   _acl = (context) ->
-    acl = []
-
-    for action, roles of context.permissions or []
-      for role in roles
-        allow = true
-        if role.indexOf('group:') is 0
-          if role == GROUP_WORLD
-            principal = EVERYONE
-          else
-            # unhandled group
-            allow = false
-            principal = role
-        else
-          if role.indexOf('acct:') is 0
-            principal = role
-          else
-            allow = false
-            principal = role
-
-        acl.push
-          allow: allow
-          principal: principal
+    parts =
+      for action, roles of context.permissions or []
+        for role in roles
+          allow: true
+          principal: role
           action: action
 
-    if acl.length
-      acl
+    if parts.length
+      acl = Array::concat parts...
     else
-      return [
-        allow: true
-        principal: EVERYONE
-        action: ALL_PERMISSIONS
-      ]
+      acl = ADMIN_PARTY
```

Comment:
a `return acl` would be nice to the end, or just return simply `ADMIN_PARTY` or `Array::contcat..`